src/test/java/com/epam/task/first/logic/ArrayLogicTest.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.task.first.logic;
+
+import com.epam.task.first.entities.Array;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static java.lang.Math.abs;
+
+public class ArrayLogicTest {
+
+    private final ArrayLogic arrayLogic = new ArrayLogic();
+
+    @Test
+    public void testFindMaxShouldFindMaxNumberInArray() {
+
+        Array array = new Array(1, 5 ,-3);
+
+        int result = arrayLogic.findMax(array);
+
+        Assert.assertEquals(5, result);
+
+    }
+
+    @Test
+    public void testFindMinShouldFindMinNumberInArray() {
+
+        Array array = new Array(1, 5, 15, 0, 79);
+
+        int result = arrayLogic.findMin(array);
+
+        Assert.assertEquals(0, result);
+
+    }
+
+    @Test
+    public void testGetAverageShouldCalculateAverageValueInArray() {
+
+        Array array = new Array(1, 2, 7);
+
+        double result = arrayLogic.getAverage(array);
+
+        double EXP = 1e6;
```

Comment:
the capital case only for class-level constants, here lowercase, but better make it class level constant with name DELTA